android/src/main/java/com/stripeterminalreactnative/ktx/TerminalExtensions.kt

```diff
@@ -98,49 +45,45 @@ suspend fun Terminal.connectReader(
     reader: Reader,
     locationId: String,
     autoReconnectOnUnexpectedDisconnect: Boolean = false,
-    listener: ReaderListenable? = null,
-    reconnectionListener: ReaderReconnectionListener
+    disconnectListener: ReaderDisconnectListener? = null,
+    reconnectionListener: ReaderReconnectionListener,
```

Comment:
Wondering if we can make `ReaderReconnectionListener` optional to pass in, since it's not required by internet and handoff readers.